nml/actions/action2var.py

```diff
@@ -557,6 +572,38 @@ def parse_via_actionD(self, expr):
     def parse_proc_call(self, expr):
         assert isinstance(expr, expression.SpriteGroupRef)
         var_access = VarAction2ProcCallVar(expr)
+        target = action2.resolve_spritegroup(expr.name)
+        refs = expr.collect_references()
+
+        # Fill param registers for the call
+        tmp_vars = []
+        for i, param in enumerate(expr.param_list):
+            if i > 0:
+                self.var_list.append(nmlop.VAL2)
+                self.var_list_size += 1
+            if refs != [expr]:
+                # With imbricated calls param registers can be overwritten
```

Comment:
imbricated?

Reply:
I couldn't find another word. What I mean is for `f(x, g(y))`, `x` can be overwritten by `y` if `f` and `g` share the same param registers.